extension/src/project/findReplace/findInProject.ts

```diff
@@ -22,7 +22,7 @@ export async function findInProject() {
     if (ProjectTreeProvider.hasProjectOpened() == false) {
         let msg = localize("autolispext.project.find.openproject", "A project must be open before you can search for a text string.");
         vscode.window.showInformationMessage(msg);
-        return;
+        return Promise.reject(msg);
```

Comment:
what will happen in a real, manual use case?

Reply:
It will not happen on manual case. So later, I will delete the test cases.